client/src/components/Markdown/Elements/HistoryLink.vue

```diff
@@ -1,6 +1,8 @@
 <template>
     <div>
-        <b-link v-if="showLink" @click="onClick">Click to Import History: {{ name }}.</b-link>
+        <b-link v-if="showLink" class="history-import-link" :data-history-id="args.history_id" @click="onClick"
```

Comment:
I think the direction the UI/UX team is moving into is moving away from classes that aren't used for applying css and instead using data attributes (https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) for test selectors. Maybe any time is a good time to also start this for selenium tests ?

Reply:
Merge this and I'll follow up with something more systematic?

Reply:
https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes

> [Custom data attributes](https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#custom-data-attribute) are intended to store custom data private to the page or application, for which there are no more appropriate attributes or elements.

I use data attributes all over the place to decorate key-value data - but assigning a ``data-component-name`` seems redundant - ``class`` would seem to be the more appropriate. ``data-component-name`` is the same as ``data-component-classification`` which we could just shorten to ``class`` 😅. But y'all attend the meetings I guess - give me the attribute to use instead of class and I'll do it.

Reply:
Briefly discussed this during the UX meeting today and realized that often `data-description` was used for this purpose. We might just want to continue doing that and add it to our best practices document for future reference. Currently there are already 23 corresponding matches in `navigation.yml`.

Reply:
I will change this but that is worse than just using the class IMO.